src/metrics/scenario.jl

```diff
@@ -180,3 +180,47 @@ end
 Calculate the mean coral evenness for each scenario for the entire domain.
 """
 scenario_evenness = Metric(_scenario_evenness, (:timesteps, :scenarios))
+
+"""
+    scenario_outcomes(rs::ResultSet, metrics::Vector{Metric})::NamedDimsArray
+
+Get outcomes for a given list of metrics and a result set
+
+# Arguments
+    - `rs` : ResultSet
+    - `metrics` : Vector of scenario Metrics (the ones that start "with scenario_")
+
+# Returns
+    NamedDimsArray with (:timesteps, :scenarios, :outcomes)
+
+# Examples
+```
+metrics::Vector{ADRIA.metrics.Metric} = [
+    ADRIA.metrics.scenario_total_cover,
+    ADRIA.metrics.scenario_asv,
+    ADRIA.metrics.scenario_absolute_juveniles,
+]
+
+# 3-dimensional Array of outcomes
+outcomes = ADRIA.metrics.scenario_outcomes(rs, metrics)
+```
+"""
+function scenario_outcomes(rs::ResultSet, metrics::Vector{Metric})::NamedDimsArray
+    num_timesteps = length(timesteps(rs))
+    num_scenarios = size(rs.inputs, 1)
+    num_metrics = length(metrics)
+
+    outcomes = NamedDimsArray(
+        zeros(Float64, num_timesteps, num_scenarios, num_metrics);
```

Comment:
`zeros()` defaults to Float64, so specifying that is unnecessary

```suggestion
        zeros(num_timesteps, num_scenarios, num_metrics);
```